core/src/test/java/org/aerogear/mobile/core/HttpServiceModuleTest.java

```diff
@@ -0,0 +1,30 @@
+package org.aerogear.mobile.core;
+
+import android.support.test.filters.SmallTest;
+
+import org.aerogear.mobile.core.http.HttpServiceModule;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(RobolectricTestRunner.class)
+@SmallTest
+public class HttpServiceModuleTest {
+
+    @Test
+    public void testGet() {
+        HttpServiceModule module = new HttpServiceModule();
+        String response = module.get("http://www.mocky.io/v2/5a5f74172e00006e260a8476");
+        assertNotNull(response);
+        assertEquals("{\n" +
```

Comment:
The content of the response should be tested, instead of the format, no? Something like:
```
Whatever response = module.get("http://www.mocky.io/v2/5a5f74172e00006e260a8476").asWhatever();
assertNotNull(response);
assertEquals(response.getStory().getTitle(), "Test Title");

Reply:
Maybe.  Right now we are verifying that the response that we expect comes back.  I've been doing this as a TDD pair programming exercise so code on the bleeding most edge would be contrived.  As things get stable I rebase them out into their own PRs for a proper review.